packages/tasks/src/dataset-libraries.ts

```diff
@@ -71,6 +71,12 @@ export const DATASET_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/argilla-io/argilla",
 		docsUrl: "https://argilla-io.github.io/argilla",
 	},
+	polars: {
+		prettyLabel: "Polars",
+		repoName: "polars",
+		repoUrl: "https://github.com/pola-rs/polars",
+		docsUrl: "https://docs.pola.rs/",
```

Comment:
should be the docs in our docs

Reply:
OK? i'll change for all the libs that we have docs for

Reply:
https://github.com/huggingface/huggingface.js/pull/825